test/spec/invertedIndexSpec.js

```diff
@@ -97,77 +85,114 @@ describe('InvertedIndex', () => {
             an: [false, true, true],
             unusual: [false, true, true],
             alliance: [false, true, true],
-          }
+          },
+          filename: 'shortStories.json'
         }
       );
     });
-    it('should return false if file index was not found', () => {
-      expect(index.getIndex('notIndex.json')).toBeFalsy();
+
+    it('correctly stores all words in the file', () => {
+      expect(index[filename].allWords).toEqual(
+        ['alice', 'falls', 'into', 'an', 'unusual', 'alliance']
+      );
     });
   });
 
-  describe('SearchIndex', () => {
-    it('should return false if filename does not exist', () => {
-      expect(index.searchIndex('alice', 'wrongData.json')).toBeFalsy();
-    });
-    it('should return return the correct search result', () => {
-      expect(typeof index.searchIndex('alice', this.filename)).toEqual('object');
-      expect(index.searchIndex('alice hahaha unusual', this.filename)).toEqual(
+  describe('GetIndex', () => {
+    it('should return the correct index', () => {
+      expect(typeof index.getIndex('shortStories.json')).toEqual('object');
+      expect(index.getIndex(filename)).toEqual(
         {
           bookTitles: ['Alice', 'Lord of the Rings', 'Rings'],
-          allWords: ['alice', 'unusual'],
+          allWords: ['alice', 'falls', 'into', 'an', 'unusual', 'alliance'],
           words: {
             alice: [true, false, false],
+            falls: [true, true, false],
+            into: [true, false, false],
+            an: [false, true, true],
             unusual: [false, true, true],
-          }
+            alliance: [false, true, true],
+          },
+          filename: 'shortStories.json'
         }
       );
     });
-    it('should return false if searchKey is an empty string', () => {
-      expect(index.searchIndex('', this.filename)).toBeFalsy();
-    });
-    it('should return false if nothing was found', () => {
-      expect(index.searchAll('abcdefgh#', this.filename)).toBeFalsy();
+
+    it('should return false if file index was not found', () => {
```

Comment:
Returning 'false' and being 'falsy' are two different things. You might want to look into this.